pages/install/firefish.md

```diff
@@ -1,16 +1,37 @@
 ---
 title: Fossil Gen 4
-deviceName: firefish
+deviceNames: [firefish, ray]
 label: ray/firefish
 section: install
-layout: aw-install
+layout: aw-install-multi
 ---
 
+<div class="callout callout-warning">
+    <h4>Variants</h4>
+    <p>The Fossil Gen 4 is marketed as different versions from different manufacturers.
+    This means that there are two known versions that are compatible with a range of Fossil Gen 4 watches.
+    These two versions will be referred to as: <code>ray</code> and <code>firefish</code></p>
+    <code>firefish</code> is compatible with the following variants:
+    <ul>
+       <li>Fossil Gen 4 46mm</li>
+       <li>Fossil Q Explorist HR</li>
+       <li>Diesel On Full Guard 2.5</li>
+       <li>Misfit Vapor 2 (46mm)</li>
+    </ul>
+    <code>ray</code> is compatible with the following variants:
+    <ul>
+       <li>Fossil Gen 4 (41mm)</li>
+       <li>Skagen Falster 2</li>
+       <li>Fossil Q Venture HR</li>
+       <li>Misfit vapor 2 (41mm)</li>
+    </ul>
+</div>
 
 <div class="callout callout-info">
     <h4>USB connection</h4>
     <p>The Fossil Gen 4 doesn't expose a USB connection. It is however possible to connect to the watch using an adapter cable.</p>
     <p>You will need access to the USB pads on the watch, this can be achieved by removing the plastic back cover. Using heat and a prying tool, it is fairly straightforward to remove. Pay attention to the flex cable that connects to the back cover to the topside of the PCB. It is recommended to start prying on the side opposite of the buttons.</p>
-    <p>Then you can solder wires to connect a USB cable to the watch. The image below indicates the function of each pad.</p>
+    <p>Then you can solder wires or use a custom dock to connect a USB cable to the watch. The image below indicates the function of each pad.</p>
```

Comment:
I'd put in a warning against soldering, as the pads are small and fragile - it's easy to mess this up and it's worth building some kind of dock

Reply:
Added a warning text below: "Be careful when soldering wires to the pads as the pads on the watch are very fragile."